src/components/ProgressBar.module.scss

```diff
@@ -0,0 +1,41 @@
+@use './../variables.scss';
+
+$initialBarWidth: 15px;
+
+.progressBar {
+  appearance: none;
+  height: 5px;
+  width: calc(100% - $initialBarWidth);
+  position: absolute;
+  top: 0;
+  left: $initialBarWidth;
+
+  // Add some initial width
+  &::after {
+    content: '';
+    height: 5px;
+    width: $initialBarWidth;
+    background-color: variables.$primary;
+    top: 0;
+    left: -$initialBarWidth;
+    position: absolute;
+  }
+
+  /* Chrome and Safari */
+  &::-webkit-progress-bar {
+    background: white;
+  }
+
+  &::-webkit-progress-value {
+    background-color: variables.$primary;
+  }
+
+  /* Firefox */
+  &::-moz-progress-bar {
+    background-color: variables.$primary;
+  }
+
+  &::-webkit-progress-value {
+    transition: width 1s ease-in-out;
+  }
```

Comment:
I was wondering why we need prefixes here. Isn't it fully supported?

https://caniuse.com/progress

Reply:
The `<progress>` element is supported but customising it is not.. so we use [these properties](https://verpex.com/blog/website-tips/how-to-style-a-progress-bar-using-css). It's also the same we do for the scrollbar in the slider component with [webkit-scrollbar](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar), for example.